app/controllers/marc_records_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Controller to handle MarcRecords
+class MarcRecordsController < ApplicationController
+  load_and_authorize_resource :organization
+  load_and_authorize_resource through: :organization
+  protect_from_forgery with: :null_session, if: :jwt_token
+
+  def show; end
+
+  def index; end
```

Comment:
Do we want to enable querying by a given `marc001`? By the latest of that `marc001` (or all unique records)?

Reply:
Or enable pagination?

Reply:
Yes, I think so... and there's maybe use cases for showing either the latest or the record from a particular file (certainly the latter for debugging data errors).

Reply:
I've added support for query by `marc001` and pagination. Lets circle about latest/sorting discussion as I think I want to better understand that.